v2/concordium/types.proto

```diff
@@ -867,6 +881,12 @@ message RejectReason {
     Empty pool_would_become_over_delegated = 53;
     // The pool is not open to delegators.
     Empty pool_closed = 54;
+    // The contract upgrade failed because of an invalid module reference
+    UpgradeInvalidModuleReference upgrade_invalid_module_reference = 55;
+    // The contract upgrade failed because the new module did not have a corresponding contract name.
+    UpgradeInvalidContractName upgrade_invalid_contract_name = 56;
+    // The contract upgrade failed because the new module does not support contract upgrades.
```

Comment:
This is not that informative. The error is more specific, namely that you tried to upgrade to an incompatible module version.